spec/services/course_exports/prepare_teams_export_service_spec.rb

```diff
@@ -40,17 +73,44 @@
   let(:coach_3) { create :faculty, school: school }
 
   let!(:school_admin) { create :school_admin, school: school }
-  let(:course_export) { create :course_export, :teams, course: course, user: school_admin.user }
+  let(:course_export) do
+    create :course_export, :teams, course: course, user: school_admin.user
+  end
+
+  let!(:team_1_reviewed_submission_1) do
+    complete_target target_l1_evaluated, student_1
+  end
 
-  let!(:team_1_reviewed_submission_1) { complete_target target_l1_evaluated, student_1 }
-  let!(:team_1_reviewed_submission_2) { complete_target target_l1_evaluated, student_1 }
-  let!(:team_2_reviewed_submission) { fail_target target_l1_evaluated, student_2 }
+  let!(:team_1_reviewed_submission_2) do
+    complete_target target_l1_evaluated, student_1
+  end
+
+  let!(:team_2_reviewed_submission) do
+    fail_target target_l1_evaluated, student_2
+  end
+
+  # Archived submission should not be present in expected data.
+  let!(:team_1_archived_submission) do
+    create :timeline_event,
+           :with_owners,
+           latest: false,
+           target: target_l1_evaluated,
+           owners: [student_1],
+           created_at: 3.days.ago,
+           archived_at: 1.day.ago
+  end
```

Comment:
The named archived submission is never referred to. This could go in the `before` block, like the other unnamed DB entries created there.

The comment applies to the entry created in `prepare_students_export_service_spec.rb` as well.

Reply:
Fixed in: https://github.com/pupilfirst/pupilfirst/commit/2d838227dc2017e5e68d1cd6c662ed188f183a14